cli/compose/convert/volume.go

```diff
@@ -42,7 +42,15 @@ func convertVolumeToMount(volumeSpec string, stackVolumes volumes, namespace Nam
 	case 1:
 		target = parts[0]
 	default:
-		return mount.Mount{}, fmt.Errorf("invald volume: %s", volumeSpec)
+		return mount.Mount{}, fmt.Errorf("invalid volume: %s", volumeSpec)
+	}
+
+	if source == "" {
```

Comment:
actuall, just thinking; should we check `len(parts)` here as well, to invalidate `-v :/foo/bar`?